src/target/kinetis.c

```diff
@@ -539,17 +545,17 @@ static bool kinetis_flash_done(target_flash_s *const f)
  * device.  This provides a fake target to allow a monitor command interface
  */
 
-static bool kinetis_mdm_mass_erase(target_s *t, platform_timeout_s *const print_progess);
```

Comment:
Why is the const disappearing here?

Reply:
`const` at that location in a function declaration (as opposed to definition) serves no purpose and is just noise - so this was removed as cleanup per the respective `clang-tidy` lint..